src/main/java/org/apache/fineract/config/WebSecurityConfiguration.java

```diff
@@ -0,0 +1,70 @@
+package org.apache.fineract.config;
+
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.enums.ParameterIn;
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeIn;
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeType;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.info.License;
+import io.swagger.v3.oas.annotations.security.SecurityScheme;
+import io.swagger.v3.oas.models.Operation;
+import io.swagger.v3.oas.models.media.StringSchema;
+import io.swagger.v3.oas.models.parameters.HeaderParameter;
+import io.swagger.v3.oas.models.parameters.Parameter;
+import org.springdoc.core.customizers.OperationCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.web.method.HandlerMethod;
+
+@OpenAPIDefinition (
+        info = @Info (
+                title = "Operations App",
+                description = "" +
+                        "Operations app is a secure, multi-tenanted microservice platform",
+                license = @License (
+                        name = "MIT Licence",
+                        url = "https://github.com/openMF/ph-ee-operations-app/blob/master/LICENSE"
+                )
+        )
+)
+@SecurityScheme (
+        name = "auth",
+        scheme = "bearer",
+        type = SecuritySchemeType.HTTP,
+        in = SecuritySchemeIn.HEADER,
+        paramName = "Authorization: Bearer",
+        description = "Use this curl request to generate authToken\n\n\n" +
```

Comment:
can we move this to text based configuration files other than hardcoding in the code?

Reply:
I am not aware of configuration done for swagger by text< according to the API spec using this method is best cause we need to change annotation also if spec gets updated and it is easier to modify for API also.